docs/manual-test-plan.md

```diff
@@ -0,0 +1,809 @@
+  --header "Authorization: Bearer $BEARER_TOKEN"
+```
+
+Verify the response has status 400, and the following json body:
+
+```json
+{
+  "success": false,
+  "reason": "Enrollment code not found"
+}
+```
+
+Finally, test the case where the user does not exist, but the enrollment code is valid:
+
+```sh
+curl -X POST 'http://local.openedx.io:8000/appsembler_api/v0/enrollment-codes/enroll-user' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "email": "doesnotexist@example.com",
+    "enrollment_code": "zbKy2BbR"
+}' \
+  --header "Authorization: Bearer $BEARER_TOKEN"
+```
+
+Verify the response has status 400 and the following json body:
+
+```json
+{
+  "success": false,
+  "reason": "User not found"
+}
+```
+
+
+### Test changing enrollment code status
+
+The enrollment-codes/status endpoint supports cancelling or restoring an enrollment code.
+
+- "cancel": invalidate the code. Also unenroll the user from the course if the code was used.
+- "restore": Make the code valid and ready to use again. Also unenroll the user from the course if the code was used.
+
+Let's restore the enrollment code that was used earlier to enroll the "apicreated4" user:
+
+
+```sh
+curl -X POST 'http://local.openedx.io:8000/appsembler_api/v0/enrollment-codes/status' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "action": "restore",
+    "enrollment_code": "PY4ngLKw"
+}' \
+  --header "Authorization: Bearer $BEARER_TOKEN"
+```
+
+Verify the response has status 200 and the following json body:
+
+```json
+{
+  "success": true
+}
+```
+
+Visit the web UI as the "apicreated4" user again,
+and verify the user is no longer enrolled in the demo course.
+
+Enroll the user in the demo course again using the same (restored) code:
+
+```sh
+curl -X POST 'http://local.openedx.io:8000/appsembler_api/v0/enrollment-codes/enroll-user' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "email": "apicreated1@example.com",
+    "enrollment_code": "PY4ngLKw"
+}' \
+  --header "Authorization: Bearer $BEARER_TOKEN"
+```
+
+Verify a success response is returned:
+
+```json
+{
+  "success": true
+}
+```
+
+In the web UI, verify the user is enrolled again in the demo course.
+
+Now cancel the enrollment code:
+
+```sh
+curl -X POST 'http://local.openedx.io:8000/appsembler_api/v0/enrollment-codes/status' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "action": "cancel",
+    "enrollment_code": "PY4ngLKw"
+}' \
+  --header "Authorization: Bearer $BEARER_TOKEN"
+```
+
+Verify the response has status 200 and the following json body:
+
+```json
+{
+  "success": true
+}
+```
+
+In the web UI, verify the user is once again no longer enrolled in the demo course.
+
+One final time, attempt to enroll the user in the course using the same code:
+
+```sh
+curl -X POST 'http://local.openedx.io:8000/appsembler_api/v0/enrollment-codes/status' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "action": "restore",
```

Comment:
This is the restoration command rather than the enrollment command. Probably copy/pasted from the wrong section.